cypress/integration/SignInValidation.spec.js

```diff
@@ -0,0 +1,11 @@
+import { SIGN_IN_PATH } from '../../src/config/paths';
+import { MEMBERS } from '../fixtures/members';
+
+describe('Name and Email Validation', () => {
+  it('Sign In', () => {
+    const { GRAASP, WRONG_EMAIL } = MEMBERS;
+    cy.visit(SIGN_IN_PATH);
+    cy.login(WRONG_EMAIL);
+    cy.login(GRAASP);
```

Comment:
I would name these commands `signInAndCheck` and `signUpAndCheck`

Reply:
Perhaps I would add some comments to give some insights on what is expected for each command (eg: `// sending wrong email show error feedback`)